### PR TITLE
enh(UI): Uniformize metric selection

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This uniformizes the metric selection in order to get the same metric selection system in both graph, top bottom and single metric

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x
- [x] 24.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by Med1233 — APPROVED** [review]

**Review by MedWassCh — DISMISSED** [review]

**Review by bdauria — DISMISSED** [review]

**Review by Med1233 — DISMISSED** [review]

**Review by Med1233 — DISMISSED** [review]

**Review by bdauria — DISMISSED** [review]

**Review by Med1233 — APPROVED** [review]

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16251,6 +16251,9 @@ msgstr "Erweiterter Modus"
 # msgid "Resource Access Management"
 # msgstr ""
 
+# msgid "You must add resources to filter on a single metric."
```
Comment (smau-centreon) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:16254]:
```suggestion
# msgid "Refine your filter on a single metric by adding more resources."
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -18345,6 +18345,9 @@ msgstr "Gestion de l'accès aux ressources"
 msgid "Refine filter"
 msgstr "Affiner le filtre"
 
+msgid "You must add resources to filter on a single metric."
+msgstr "Vous devez ajouter des ressources afin de filtrer sur une seule métrique."
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18349]:
```suggestion
msgstr "Affinez votre filtre sur une seule métrique en ajoutant d'autres ressources."
```

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -16584,6 +16584,9 @@ msgstr "Modo avanzado"
 # msgid "Resource Access Management"
 # msgstr ""
 
+# msgid "You must add resources to filter on a single metric."
```
Comment (smau-centreon) [inline centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:16587]:
```suggestion
# msgid "Refine your filter on a single metric by adding more resources."
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -18345,6 +18345,9 @@ msgstr "Gestion de l'accès aux ressources"
 msgid "Refine filter"
 msgstr "Affiner le filtre"
 
+msgid "You must add resources to filter on a single metric."
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18348]:
```suggestion
msgid "Refine your filter on a single metric by adding more resources."
```

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -17048,6 +17048,9 @@ msgstr ""
 # msgid "Resource Access Management"
 # msgstr ""
 
+# msgid "You must add resources to filter on a single metric."
```
Comment (smau-centreon) [inline centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:17051]:
```suggestion
# msgid "Refine your filter on a single metric by adding more resources."
```

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -17037,6 +17037,9 @@ msgstr "Você não salvou as alterações, deseja continuar?"
 # msgid "Resource Access Management"
 # msgstr ""
 
+# msgid "You must add resources to filter on a single metric."
```
Comment (smau-centreon) [inline centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:17040]:
```suggestion
# msgid "Refine your filter on a single metric by adding more resources."
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by kduret — APPROVED** [review]
ok for cypress e2e changes

**Review by smau-centreon — APPROVED** [review]

**Review by Med1233 — APPROVED** [review]

**Pull request merged**